jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -86,6 +86,14 @@ public QuickStartConfiguration()
         addDependents(WebXmlConfiguration.class);
     }
 
+    private boolean isQuickStartEnabled(WebAppContext context)
```

Comment:
I don't see the usefulness of the context attribute? Nor why if it's not set, this method returns `true`?

Reply:
The `QuickStartConfiguration` is shared across contexts.
The field `_quickStart` is the wrong way to track this behavior.

Reply:
Also removed the `_mode` field as it's duplicates information that's already present in the context but in the wrong way.